grunt/tasks/dist.js

```diff
@@ -1,5 +1,5 @@
 // grunt/tasks/dist.js
 
 module.exports = function(grunt) {
-  grunt.registerTask('dist', ['assemble:dist', 'scsslint', 'sass:dist','postcss:dist', 'browserify:dist', 'accessibility', 'groc', 'modernizr:dist', 'copy:dist', 'browserSync:dist']);
-};
\ No newline at end of file
+  grunt.registerTask('dist', ['assemble:dist', 'scsslint', 'sass:dist','postcss:dist', 'browserify:dist', /*'accessibility',*/ 'groc', 'modernizr:dist', 'copy:dist', 'copy:quilt_file', 'copy:quilt_assets', 'browserSync:dist']);
```

Comment:
The accessibility was testing many things, but not the HTML generated by the JavaScript. When running the "dist" task, I didn't want unnecessary warnings and errors that make it appear to have not worked.

The accessibility will still be run on `grunt test`